aries_cloudagent/multitenant/manager.py

```diff
@@ -170,7 +185,9 @@ async def create_wallet(
             # In unmanaged mode we don't want to store the wallet key
             if key_management_mode == WalletRecord.MODE_UNMANAGED:
                 del settings["wallet.key"]
-
+            # Adding wallet_webhook_urls and wallet_dispath_type to wallet_record
+            settings["wallet.webhook_urls"] = wallet_webhook_urls
```

Comment:
This is redundant. How about `wallet_webhook_urls` and `wallet_dispatch_type` add at `settings` in wallet_create of route.py. So. we can remove these parameters in create_wallet function.

Reply:
I agree! We can store `wallet.webhook_urls` and `wallet.dispatch_type` inside the settings of the wallet record. 

Only thing we need to do then is modify `admin.webhook_urls` based on `wallet.dispatch_type`

Reply:
Updated implementation based upon feedback, now storing these attributes inside the settings of the wallet record and modifying admin.webhook_urls based upon dispatch_type.